contributing/mo.s.md

```diff
@@ -1,4 +1,4 @@
-# [Standard] Method of Operation
+# [MO] Method of Operation
```

Comment:
It's the Standard for writing a method of operation :) (so there shouldn't be any change here)